database/postgres/dal/league.py

```diff
@@ -59,3 +60,9 @@ def delete_league(self, league_id: int) -> bool:
         self.db_session.delete(league)
         self.db_session.commit()
         return True
+
+    def get_query(self):
```

Comment:
You add these two methods everywhere and they work exactly the same. Maybe create a base class for our DALs which would implement those methods and then just inherit it?